src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -61,7 +61,7 @@
 (def ProtocolandDependenciesMap
   "Schema defining the map used to represent the protocol and dependencies
   of a service."
-  {:fns [clojure.lang.PersistentList]
+  {:fns (schema/pred seq?)
```

Comment:
So this just checks that `fns` is a sequence, rather than a sequence of sequences.  I think you want it to be `[(schema/pred seq?)]`?